site/content/docs/5.3/utilities/borders.md

```diff
@@ -52,6 +52,7 @@ Change the border color using utilities built on our theme colors.
 {{< /border.inline >}}
 <span class="border border-black"></span>
 <span class="border border-white"></span>
+<span class="border border-tertiary"></span>
```

Comment:
It's not used in this PR nor in other PRs created so far. I suggest mentioning it in the dark mode PR description for now until we find a use case where it's actually needed.